src/data_f.c

```diff
@@ -209,6 +210,138 @@ FC_SHMEM_PUTMEM(void *target, void *source, fortran_integer_t *len, fortran_inte
 }
 
 
+#define FC_SHMEM_CHARACTER_PUT_NBI FC_FUNC_(shmem_character_put_nbi, SHMEM_CHARACTER_PUT_NBI)
+void FC_SHMEM_CHARACTER_PUT_NBI(void *dest, void *source, fortran_integer_t *nelems, fortran_integer_t *pe);
+void
+FC_SHMEM_CHARACTER_PUT_NBI(void *dest, void *source, fortran_integer_t *nelems, fortran_integer_t *pe)
+{
+    SHMEM_ERR_CHECK_INITIALIZED();
+
+    shmem_internal_put_nb(dest, source, SIZEOF_FORTRAN_CHARACTER * *nelems, *pe, NULL);
+}
```

Comment:
Ugh, this may be the case.  I'm going to log an issue for it, since the issue was propagated, but not introduced in 1.3.